.github/templates/tilgangsstyring-pnd.md

```diff
@@ -66,3 +66,49 @@ labels: 'kind/deploy_patch, team/tilgangsstyring'
 [^1]: Approve pending prod releases by clicking the blue production chip and clicking approve. ![image-20240306133832594](https://raw.githubusercontent.com/Altinn/altinn-authorization/main/.github/images/ado-pending-approval-screen.png)
 [^2]: Look for any blue production chips. ![image-20240306133137061](https://raw.githubusercontent.com/Altinn/altinn-authorization/main/.github/images/ado-prod-button.png)
 [^3]: Look for any blue TT02 chips. ![tt02-button](https://raw.githubusercontent.com/Altinn/altinn-authorization/main/.github/images/ado-tt02-button.png)
```

Comment:
Footnotes/links should generally be at the end of the file.

Reply:
Moved them to the end of the file.